src/Mizar/SamplingWithFrameTrackInputWidget.h

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef MIZAR_SAMPLING_WITH_FRAME_TRACK_INPUT_WIDGET_H_
+#define MIZAR_SAMPLING_WITH_FRAME_TRACK_INPUT_WIDGET_H_
+
+#include <QWidget>
+#include <memory>
+#include <string_view>
+
+#include "MizarData/MizarPairedData.h"
+
+namespace Ui {
+class SamplingWithFrameTrackInputWidget;
+}
+
+namespace orbit_mizar {
+
+template <typename PairedData>
+class SamplingWithFrameTrackInputWidgetTmpl : public QWidget {
+ public:
+  explicit SamplingWithFrameTrackInputWidgetTmpl(QWidget* parent);
+  ~SamplingWithFrameTrackInputWidgetTmpl() override;
+
+  void Init(const PairedData* data, const QString& name);
```

Comment:
What about giving that function a more descriptive name. Something like `SetSamplingData` or so?

Reply:
`Init` name makes the user think "unless I call the method, an instance of the class is useless", `SetWhatever` -- not so much. Also, the method signature clearly states what exactly we set, doesn't it?